plugins/modules/database/influxdb/influxdb_retention_policy.py

```diff
@@ -156,12 +169,28 @@ def alter_retention_policy(module, client, retention_policy):
     elif duration == 'INF':
         influxdb_duration_format = '0'
 
+    if shard_group_duration:
+        shard_group_duration_lookup = duration_regexp.search(shard_group_duration)
+        if not shard_group_duration_lookup:
+            module.fail_json(
+                msg="Failed to parse value of shard_group_duration. Please see the documentation for valid values")
```

Comment:
The documentation does not specify which values are valid. There is only one example (`1h`) and that's it.